loops_lab.js

```diff
@@ -0,0 +1,270 @@
+// # Loops Lab
```

Comment:
Overall good demonstration and implementation of for and while loops. Other than minor issue with variable declaration for for loops, everything else looks fine. Keep up the good work!